common/config.h

```diff
@@ -70,6 +71,7 @@
 
 //file names
 #define TDNF_REPO_METADATA_MARKER         "lastrefresh"
+#define TDNF_REPO_METADATA_MIRRORLIST     "mirrorlist"
```

Comment:
is it okay to do:
`#define TDNF_REPO_METADATA_MIRRORLIST TDNF_REPO_KEY_MIRRORLIST`
and keep them together?

Reply:
These are two different things, they just happen to have the same value. One is a config setting key, the other is a file name. They could be different.

Reply:
In https://github.com/vmware/tdnf/pull/479 you have done the opposite of this :)

Reply:
It's not the same pattern. There we have an option common in both main config and repos, therefore they should have the same value. Here, we could just as well have a different value, like "mirror.list" for the file name.